website/source/docs/secrets/pki/index.html.md

```diff
@@ -579,6 +1097,12 @@ If you get stuck at any time, simply run `vault path-help pki` or with a subpath
         `ec` keys. See https://golang.org/pkg/crypto/elliptic/#Curve
         for an overview of allowed bit lengths for `ec`.
       </li>
+      <li>
+        <span class="param">use_csr_common_name</span>
```

Comment:
I can default it to true, but the use case is to use the same set of options regardless of whether you're issuing a new cert/private key or issuing based on a CSR.